src/python_testing/TC_IDM_1_4.py

```diff
@@ -74,7 +74,7 @@ async def test_TC_IDM_1_4(self):
         cap_for_batch_commands = 100
         number_of_commands_to_send = min(max_paths_per_invoke + 1, cap_for_batch_commands)
 
-        invalid_command_id = 0xffff_ffff
+        invalid_command_id = 0xfff4_00ff
         list_of_commands_to_send = []
         for endpoint_index in range(number_of_commands_to_send):
```

Comment:
In general I would like to understand, are you having issues on the device receiving this command or the device sending the command?

If you are having issues receiving the command on the server, then I think you are not conforming to the spec. My understanding is that "8.8.2.3. Incoming Invoke Request Action" is intended to be followed in the order stated. You are supposed to check that:

```
If this action contains more CommandDataIB elements in the InvokeRequests list than are
supported by the device (see MaxPathsPerInvoke), then a Status Response action with the
INVALID_ACTION Status Code SHALL be submitted to the message layer and this interaction
SHALL terminate.
```

Before even looking at any of the requests and their contents